deviceModeInit.js

```diff
@@ -287,7 +408,12 @@ var rudderTracking = (function () {
         }
       } else {
         if (payload[j.src]) {
```

Comment:
Any chance of `j` not being present inside payload? If we could error-handle those cases.

// non-blocking

Reply:
j is just working as a reference for dest and source key mapping [here](https://github.com/rudderlabs/rudder-shopify-tracker/blob/ddf7b3459a2ac2f706a6f6fb5a278214aaa0caf4/deviceModeInit.js#L14)